### DISCUSSION TITLE
Reset the current local env + cluster env

### DISCUSSION THREAD

**Discussion by 0x53fa29jhezc** [fluxcd/flux2#453] [Q&A] [answered]
I messed up my local env 

The command 

flux get helmreleases -A

prints me a release that I have already deleted 

I tried brew
- flux uninstall 
- brew uninstall Flux -> then reinstall
- i have created a new cluster and also  bootstraped with flux it in a new folder
- i have deleted ~/.kube/ 

How can I get the flux env in like "factory setting"?

**Comment by kingdonb** [comment, marked as answer]
I think there is more than one way that HelmRelease resources can get stuck. What is happening generally speaking is that Helm Controller wants to clean up after itself, it does some work to guarantee that it can clean up before the resource is terminated in the form of adding a "Finalizer" to the resource.

If you edit the HelmRelease resource after it is submitted to the cluster, you find a section in the metadata for finalizers:

```
metadata:
  finalizers:
  - finalizers.fluxcd.io
```

Set finalizers to `null` to allow Kubernetes to delete the release. This means that Helm Controller will not do any cleaning up of its own, but as you said you've already cleaned up the Helm release with `helm uninstall` I assume or some other way.

Hope this helps!
